src/utilities/string_functions.js

```diff
@@ -222,6 +222,9 @@ p5.prototype.matchAll = function(str, reg) {
  * then unused decimal places will be set to 0. For example, calling
  * `nf(123.45, 4, 3)` returns the string `'0123.450'`.
  *
+ * When the number is negative, for example, calling `nf(-123.45, 5, 2)`
```

Comment:
Thank you for adding that. Just one last thought: could you also modify the reference example so that it appears like this?

![image](https://github.com/user-attachments/assets/bf02ac15-d92c-4aa0-acf1-e29507b265b6)

Since we are not counting the minus sign as an extra digit, negative numbers may have greater width due to the additional minus sign. Therefore, we could show users that negative numbers occupy more space as we are not counting it.

Reply:
ya no problem.